### PR TITLE
Add topic page proto and config; Read the config on server start

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/topic_page.proto [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  bool show_decrease = 4;
+  string diff_base_date = 5; // For comparisons
+}
+
+message HighlightMetadata {
+  string description = 1;
+}
+
+message Tile {
+  enum TitleType {
```
Comment (chejennifer) [inline server/topic_page.proto:53]: TitleType -> TileType?

**Review by chejennifer — COMMENTED** [review]
Nice!

**Review by chejennifer — APPROVED** [review]

File: server/topic_page.proto (hunk 1 above)
Comment (shifucun) [reply server/topic_page.proto]: Fixed

**Comment by shifucun** [comment]
Thanks for review!

File: server/topic_page_config.textproto [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 Google LLC
```
Comment (beets) [inline server/topic_page_config.textproto:1]: we'll have a set of these, one per topic x country, so would be nice to move them into a separate folder as we add more config files.

**Review by beets — COMMENTED** [review]
yay, protos :)

**Pull request merged**